src/headers/wazuhdb_op.h

```diff
@@ -19,22 +19,52 @@
 #define FIELD_SEPARATOR_DBSYNC "|"
 #define FIELD_SEPARATOR_DBSYNC_ESCAPE "\uffff"
 
+
+/// Enumeration of communication with Wazuh DB components.
+typedef enum
+{
+    SYSCOLLECTOR_PROCESSES,
+    SYSCOLLECTOR_PACKAGES,
+    SYSCOLLECTOR_HOTFIXES,
+    SYSCOLLECTOR_PORTS,
+    SYSCOLLECTOR_NETWORK_PROTOCOL,
+    SYSCOLLECTOR_NETWORK_ADDRESS,
+    SYSCOLLECTOR_NETWORK_IFACE,
+    SYSCOLLECTOR_HWINFO,
+    SYSCOLLECTOR_OSINFO,
+    SCAN_INFO_GET,
+    UPDATEDATE,
+    CLEANDATABASE,
+    SCAN_INFO_UPDATE,
+    CONTROL,
+    LOAD,
+    COMP_DELETE,
+    SAVE,
+    SAVE2,
+    COMP_INTEGRITY_CLEAR,
+    COMPONENT_INVALID // Fallback for invalid values
+} ComponentType;
```

Comment:
We could use `component_type` to be consistent with the naming of the other enum

Reply:
Done!